Makefile

```diff
@@ -59,7 +59,7 @@ endif
 
 
 ifeq ($(BUILD_OMS),BUILD_OMS)
-dsckit098: nx nxOMSAgent nxOMSPlugin nxOMSCustomLog nxFileInventory nxOMSGenerateInventoryMof
+dsckit098: nx nxOMSAgent nxOMSPlugin nxOMSCustomLog nxFileInventory nxOMSGenerateInventoryMof nxOMSAutomationWorker
```

Comment:
Correct. Setup for DIY is handled by the user - after getting binaries from service